include/common/path.h

```diff
@@ -5,6 +5,14 @@
 
 namespace BoBRobotics {
 namespace Path {
+//! Get the path of the folder containing the currently running program
+filesystem::path
+getProgramFolder();
```

Comment:
``getProgramDirectory`` might be a better name - folder always feels a bit 90s

Reply:
True... I must have been feeling nostalglic for Windows 95 for a moment.

Reply:
Why do you think XFCE is my favourite window manager....